src/Auth0Client.ts

```diff
@@ -457,9 +469,16 @@ export default class Auth0Client {
 
     await this.cacheManager.set(cacheEntry);
 
-    this.cookieStorage.save('auth0.is.authenticated', true, {
+    this.cookieStorage.save(COOKIE_IS_AUTHENTICATED_HINT, true, {
       daysUntilExpire: this.sessionCheckExpiryDays
     });
+
+    if (decodedToken.claims.org_id) {
+      this.cookieStorage.save(
+        this.orgHintCookieName,
+        decodedToken.claims.org_id
+      );
+    }
```

Comment:
If there's no `org_id` claim it means that the user did not sign in in the context of an organization, so IMO it makes sense to remove the cookie so that we don't try to use it the next time.

Reply:
Yes, I was asking because in the above case, we are not removing it.

Reply:
You're right, I've added this now in https://github.com/auth0/auth0-spa-js/pull/788/commits/98dda60613179ac2a49d4c55f9966c1eaede551d